ASB MAY 2025 Security Patches integration

AlamIntel opened the pull request:
Integrating Google Android Security Bulletin Patches.

Test done: All STS r32 TCs Passed Except 2 TC
CVE_2024_0014 (issue id:133100)
CVE_2016_6786 (issue id:133098)

Tracked-On: OAM-132697

AlamIntel committed:
ASB MAY 2025 Security Patches integration

Integrating Google Android Security Bulletin Patches.

Test done: STS r38 TCs Passed

Tracked-On: OAM-131817
Signed-off-by: Alam, Sahibex <sahibex.alam@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

AlamIntel committed:
ASB JUN 2025 Security Patches integration

Integrating Google Android Security Bulletin Patches.

Test done: All STS r32 TCs Passed Except 2 TC
CVE_2024_0014 (issue id:133100)
CVE_2016_6786 (issue id:133098)

Tracked-On: OAM-132697
Signed-off-by: Alam, Sahibex <sahibex.alam@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
FAILURE: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

tanujtekriwal reviewed (approved):
LGTM

shyjumon-n reviewed (approved):
LGTM

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is FAILURE. Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is FAILURE. Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon_tmr0_master-merge/118
